packages/core/src/common/_variables.scss

```diff
@@ -51,32 +53,31 @@ $pt-font-family:
 
 $pt-font-family-monospace: monospace !default;
 
-$pt-font-size: $pt-grid-size * 1.4 !default;
-$pt-font-size-large: $pt-grid-size * 1.6 !default;
-$pt-font-size-small: $pt-grid-size * 1.2 !default;
+$pt-font-size: $bp-spacing * 3.5 !default;
+$pt-font-size-large: $bp-spacing * 4 !default;
+$pt-font-size-small: $bp-spacing * 3 !default;
 
 // a little bit extra to ensure the height comes out to just over 18px (and browser rounds to 18px)
-// equivalent to: math.div($pt-grid-size * 1.8, $pt-font-size) + 0.0001
+// equivalent to: math.div($bp-spacing * 4.5, $pt-font-size) + 0.0001
 $pt-line-height: 1.28581 !default;
 
 // Grids & dimensions
 
-// equivalent to: floor(math.div($pt-grid-size, 5))
-$pt-border-radius: 2px !default;
+$pt-border-radius: $bp-spacing * 0.5 !default;
```

Comment:
Should the radius depend on spacing automatically? I feel like these are separate design decisions that aren't necessarily coupled (e.g. I could imaging increasing bp-spacing to 6px, but that doesn't mean border-radius has to go up to 3px).

Reply:
Fair point. I'll revert this particular change for now. I think it would make sense to define multiple border-radius sizes at some point (e.g. sm, md, lg) to gain more consistency, but we don't need to bundle that change with this one. 793b6fbadaf2458ac566cdb1468c7d651e086bfa